code/game/objects/structures/industrial_lift.dm

```diff
@@ -496,7 +499,7 @@ GLOBAL_DATUM(central_tram, /obj/structure/industrial_lift/tram/central)
 		return PROCESS_KILL
 	else
 		travel_distance--
-		lift_master_datum.MoveLiftHorizontal(travel_direction, z)
+		lift_master_datum.MoveLiftHorizontal(travel_direction, z, DELAY_TO_GLIDE_SIZE(SStramprocess.wait))
```

Comment:
DELAY_TO_GLIDE_SIZE scales based on the delta of real time and byond time. Does that actually fit what you want here? Since we don't care about the whole system, just tram process

Reply:
There's a glide_size_multiplier global var which is updated along with time dilatation by the time track subsystem, so yea, I guess it's a good thing to have and probably also helps mitigating tram stutter.

Reply:
Couldn't it possibly undercorrect for tram's stutter? since we're tracking the delta of everything, rather then just the tram

Reply:
Well, it's meant to mitigate the impact of server lag on the visual flow of movement. You might be right buuuuuut it's too hard to tell on a local server where not a whole lot of thing are going on and the server lag is fairly low. Futhermore, I lack the means to test that with my subpar hardware.

Reply:
We'll stress it on live. Keep an eye on this yeah?